pkg/mutation/system_external_data_test.go

```diff
@@ -274,12 +336,42 @@ func TestSystem_resolvePlaceholders(t *testing.T) {
 			},
 		},
 	}
+	var clientCertWatcher *certwatcher.CertWatcher
+	clientCertFile, err := ioutil.TempFile("", "client-cert")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Remove(clientCertFile.Name())
+
+	_, _ = clientCertFile.WriteString(clientCert)
```

Comment:
would we want to check error for WriteString?

Reply:
I skipped the error checking since it's a test but I can add one.